misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -3007,12 +3008,14 @@ public ReceiveActionNode transform(ReceiveActionNode receiveActionNode) {
 
     @Override
     public ReceiveFieldsNode transform(ReceiveFieldsNode receiveFieldsNode) {
-        Token openBrace = formatToken(receiveFieldsNode.openBrace(), 0, 1);
+        int trailingNL = shouldExpand(receiveFieldsNode) ? 1 : 0;
+        Token openBrace = formatToken(receiveFieldsNode.openBrace(), 0, trailingNL);
         indent();
-        SeparatedNodeList<NameReferenceNode> receiveFields = formatSeparatedNodeList(receiveFieldsNode.receiveFields(),
-                0, 1, 0, 1);
-        Token closeBrace = formatToken(receiveFieldsNode.closeBrace(), 0, 1);
+        SeparatedNodeList<Node> receiveFields =
+                formatSeparatedNodeList(receiveFieldsNode.receiveFields(), 0, 0, 1 - trailingNL, trailingNL, 0,
```

Comment:
can `1 - trailingNL` be negative?

Reply:
This cannot be because the maximum value of `trailingNL` is 1. So this is either 1 or 0. (invert of `trailingNL`) https://github.com/poorna2152/ballerina-lang/blob/multiple_receive_formatter/misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java#L3011